### ISSUE TITLE
Layout request

### ISSUE THREAD

**Issue by gotdoug** [TomAnthony/itermocil#101]
Hello, I have run into a situation, where I would like to be able to do a layout like either of these:
`
.------------------.------------------.
| (0)        | (1)        | (2)       |
|            |            |           |
|-------------------------------------|
| (3)              | (4)              |
|                  |                  |
|                  |                  |
|-------------------------------------|
| (5)              | (6)              |
|                  |                  |
|                  |                  |
'------------------'------------------'

.------------------.------------------.
| (0)              | (1)              |
|                  |                  |
|                  |                  |
|-------------------------------------|
| (2)              | (3)              |
|                  |                  |
|                  |                  |
|-------------------------------------|
| (4)        | (5)        | (6)       |
|            |            |           |
'------------------'------------------'
`

Basically, have a single row of 3 columns, and the all the remaining terminals in rows of 2 columns, what whatever number is needed.

Is that something that can be done with currently layouts, that I am just missing? If not, would it be difficult to implement? Unfortunately, I don't have the skills to implement this myself, at this time.
